admin/templates/configuration/bin/qserv-restart.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+{{QSERV_RUN_DIR}}/bin/qserv-stop.sh
+{{QSERV_RUN_DIR}}/bin/qserv-start.sh
+{{QSERV_RUN_DIR}}/bin/qserv-status.sh
```

Comment:
"restart" usually means "stop + start", not "stop + start + status", if you need to run qserv-status because qserv-start does not shown some info then maybe qserv-start needs to be modified to display that info.

I would probably make qserv-start conditional and only run it if qserv-stop returns OK code like:

    {{QSERV_RUN_DIR}}/bin/qserv-stop.sh && {{QSERV_RUN_DIR}}/bin/qserv-start.sh